src/status_im/ui/screens/wallet/settings/events.cljs

```diff
@@ -0,0 +1,20 @@
+(ns status-im.ui.screens.wallet.settings.events
+  (:require [status-im.ui.screens.accounts.events :as accounts]
+            [status-im.utils.ethereum.core :as ethereum]
+            [status-im.utils.handlers :as handlers]))
+
+(defn- toggle-checked [ids id checked?]
+  (if checked?
+    (conj ids id)
+    (disj ids id)))
+
+(handlers/register-handler-fx
+  :wallet.settings/toggle-visible-token
```

Comment:
this should be `handlers/register-handler-db` instead of `handlers/register-handler-fx` - with the later, you have to return effects map (where `db` effect is associated under `:db` key).

Reply:
Yes this is the case. You actually wrote the function!